test/test_impl.py

```diff
@@ -130,29 +128,54 @@ def test_verify_github_attested(self) -> None:
         bundle = Bundle.from_json(gh_signed_dist_bundle_path.read_bytes())
         attestation = impl.Attestation.from_bundle(bundle)
 
-        predicate_type, predicate = attestation.verify(verifier, pol, gh_signed_dist)
+        predicate_type, predicate = attestation.verify(pol, gh_signed_dist)
         assert predicate_type == "https://docs.pypi.org/attestations/publish/v1"
         assert predicate == {}
 
+    @pytest.mark.parametrize("claims", (None, {}, {"ref": "refs/tags/v0.0.4a2"}))
```

Comment:
I have mixed feelings on that -- on one hand we could use the `claims` to refine the verification, and on the other IMO the difference there is harder to explain to users.

This goes back to the discussion we were having about claims more generally -- IMO their presence at all was a bit of a mistake in my writing of PEP 740, and we might be better off just ignoring them entirely since they're technically not part of the trusted publisher identity at all.